charger/go-e/api.go

```diff
@@ -78,20 +82,25 @@ func (c *LocalAPI) response(partial string, res interface{}) error {
 }
 
 // Status reads a v1/v2 api response
-func (c *LocalAPI) Status() (Response, error) {
-	if c.v2 {
-		res := new(StatusResponse2)
-		err := c.response("status?filter=alw,car,eto,nrg,wh,trx,cards", &res)
-		return res, err
+func (c *LocalAPI) Status() (res Response, err error) {
+	if time.Since(c.updated) > c.cache {
```

Comment:
Verstehe ich immer noch nicht. Warum musste localapi das vorher nicht zu unterscheiden und du musst das jetzt?

Reply:
Hä? Im originalen Code wird oben nach v2 geprüft und Responsev2 geliefert und die Funktion innerhalb des If verlassen. Wenn V2 nicht zutrifft wurde der untere Code ausgeführt. Es wurde im Alten Code unterschieden und im neuen Code wird auch unterschieden. Warum es 2 unterschiedliche Responses gibt, weiß ich nicht. Das habe ich so nicht gebaut, das war schon vorher so.
Ich habe jetzt keinen Bock mehr, hier über Trivialitäten und leerzeilen zu diskutieren, für solche Spielchen ist mir echt die Zeit zu schade. Nimm es oder rejecte es. Ich nutze dann halt meinen Fork. Ich wollte nur unterstützen, aber sowas habe ich noch in keinem Open Source Projekt erlebt